### ISSUE TITLE
What is the sort used in the "Contributed to" org list?

### ISSUE THREAD

**Issue by philderbeast** [AurelienLourot/ghuser.io#142]
On https://ghuser.io/philderbeast I see the contributions list and agree with the sort order. What is the sort used in the "Contributed to" org list? It doesn't seem to parallel the contributions list.

**Comment by AurelienLourot** [comment]
**TL;DR** it's kind of random

**Details:** when your profile gets created, all the repos you have contributed to are appended to a list from the newest contribution to the oldest and the "Contributed to" org list is produced by iterating on this list, so it's as well *kind of* from the newest to the oldest. But then every day your profile gets refreshed and new contributions get appended to this list, so it ends up looking like "D C B A E F G" (where A is the oldest and G the newest). No effort has been made so far to sort this list in a meaningful way, so it's not random, but as good as random.

**Comment by philderbeast** [comment]
Early days yet. Thanks for creating this.
